ui/shared/components/panel/sample.jsx

```diff
@@ -16,39 +16,40 @@ const iconColor = (loadedSample, isOutdated) => {
   return isOutdated ? 'grey' : 'green'
 }
 
-const Sample = React.memo(({ loadedSample, isOutdated, hoverDetails }) => (
+const Sample = React.memo(({ sampleType, datasetType, loadedDate, hoverContent, isOutdated, hoverDetails }) => (
```

Comment:
I refactored this component to work with passing all the sample props through instead of requiring an object representation of the sample, so that for displaying phenotype priority and RNA we can explicitly pass in the fields we want instead of needing to format the data as if it were a search. sample

Reply:
🙏 thanks for doing this, I always felt like I was using this component in hackyish ways for rna and phenotype priority but refactoring was out of scope